src/sagemaker/jumpstart/utils.py

```diff
@@ -63,13 +63,64 @@ def get_jumpstart_launched_regions_message() -> str:
     return f"JumpStart is available in {formatted_launched_regions_str} regions."
 
 
+def get_jumpstart_private_content_bucket(
+    region: str = constants.JUMPSTART_DEFAULT_REGION_NAME,
+) -> str:
+    """Returns regionalized private content bucket name for JumpStart.
+
+    Raises:
+        ValueError: If JumpStart is not launched in ``region`` or private content
+            unavailable in that region.
+    """
+
+    old_private_content_bucket: Optional[
+        str
+    ] = accessors.JumpStartModelsAccessor.get_jumpstart_private_content_bucket()
+
+    info_logs: List[str] = []
+
+    private_bucket_to_return: Optional[str] = None
+    if (
+        constants.ENV_VARIABLE_JUMPSTART_CONTENT_BUCKET_OVERRIDE in os.environ
+        and len(os.environ[constants.ENV_VARIABLE_JUMPSTART_CONTENT_BUCKET_OVERRIDE]) > 0
+    ):
+        private_bucket_to_return = os.environ[
+            constants.ENV_VARIABLE_JUMPSTART_CONTENT_BUCKET_OVERRIDE
+        ]
+        info_logs.append(f"Using JumpStart private bucket override: '{private_bucket_to_return}'")
+    else:
+        try:
```

Comment:
I think it would be cleaner to use `os.environ.get` and then throw a value error if the result is `None`